x-pack/plugins/canvas/canvas_plugin_src/functions/common/font.ts

```diff
@@ -63,14 +63,14 @@ export function font(): NullContextFunction<'font', Arguments, Style> {
         types: ['boolean'],
       },
       lHeight: {
-        aliases: ['lineHeight'],
+        aliases: ['lineHeight', 'null'],
```

Comment:
I think `null` was supposed to be added to `types` instead of `aliases`?

Reply:
Nice catch! I meant to put to it on types. I'll fix it.